hasher-matcher-actioner/terraform/main.tf

```diff
@@ -226,6 +226,16 @@ module "durable_fs" {
   additional_tags = var.additional_tags
 }
 
+module "custodian" {
```

Comment:
This is a very specific custodian. Let's use "lcc_custodian" while in terraform/main.tf.